src/lib/rust.ts

```diff
@@ -215,8 +216,8 @@ type InstallCargoDependencyArg = {
   toolchain: string | undefined;
   name: string;
   version: string;
-  dependencyTmpDirPath: string;
   dependencyDirPath: string;
+  dependencyTmpDirPath: string;
```

Comment:
please move it back, just to remove additional change that does nothing. Thanks!